index.js

```diff
@@ -0,0 +1,45 @@
+const { test } = require('mocha')
+
+function validatePassword(password) {
+  const lowerCase = 'abcdefghijklmnopqrstuvwxyz'
+  const upperCase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+  const specialChar = '!@#$%^&*()<>?_+}{-=[]\|,./`~'
+  const listNumbers = '0123456789'
+
+  let checkLength = false
+  let checkLowerCase = true
+  let checkSpecialChar = true
+  let checkUpperCase = true
+  let checkNumber = true
+
+  function passwordLength() {
+    if (password.length < 8) {
+      // empty
+    }
+  }
+
+  function passwordLowerCase(lowerCase) {
+    if (password.includes(checkLowerCase)) {
```

Comment:
You are passing in true or false?

Do you mean to pass in "lowercase" in the if statement? 

Actually you should be doing (psuedo code):

For loop on password (let i){
   If Lowercase.includes(Password[i]){
      Checklowercase = true
   }
}